[2022-03-08T17:44:03.123920+01:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/another-page"" at /home/cyril/Bureau/blog/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/another-page\" at /home/cyril/Bureau/blog/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/another-page/\". at /home/cyril/Bureau/blog/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2022-03-08T17:44:03.296176+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-03-08T17:44:03.298153+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-03-08T17:44:03.528752+01:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9f1bfd"},"request_uri":"http://127.0.0.1:8000/_wdt/9f1bfd","method":"GET"} []
[2022-03-08T17:44:13.503792+01:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/"" at /home/cyril/Bureau/blog/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/\" at /home/cyril/Bureau/blog/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /home/cyril/Bureau/blog/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:186)"} []
[2022-03-08T17:44:13.506384+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-03-08T17:44:13.602243+01:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"564c03"},"request_uri":"http://127.0.0.1:8000/_wdt/564c03","method":"GET"} []
[2022-03-08T17:44:13.635946+01:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/favicon.ico" (from "http://127.0.0.1:8000/")" at /home/cyril/Bureau/blog/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/favicon.ico\" (from \"http://127.0.0.1:8000/\") at /home/cyril/Bureau/blog/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /home/cyril/Bureau/blog/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2022-03-08T17:44:13.664651+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-03-08T17:44:13.665830+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-03-09T08:18:09.510686+01:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/liste-des-articles"" at /home/cyril/Bureau/blog/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/liste-des-articles\" at /home/cyril/Bureau/blog/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/liste-des-articles/\". at /home/cyril/Bureau/blog/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2022-03-09T08:18:09.725433+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-03-09T08:18:09.726850+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-03-09T08:18:09.975890+01:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"4080f9"},"request_uri":"http://127.0.0.1:8000/_wdt/4080f9","method":"GET"} []
[2022-03-09T08:18:18.049534+01:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/"" at /home/cyril/Bureau/blog/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/\" at /home/cyril/Bureau/blog/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /home/cyril/Bureau/blog/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:186)"} []
[2022-03-09T08:18:18.052076+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-03-09T08:18:18.134861+01:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"e4df19"},"request_uri":"http://127.0.0.1:8000/_wdt/e4df19","method":"GET"} []
[2022-03-09T08:18:18.154228+01:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/favicon.ico" (from "http://127.0.0.1:8000/")" at /home/cyril/Bureau/blog/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/favicon.ico\" (from \"http://127.0.0.1:8000/\") at /home/cyril/Bureau/blog/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /home/cyril/Bureau/blog/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2022-03-09T08:18:18.183194+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-03-09T08:18:18.184566+01:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-03-09T09:08:08.651538+01:00] console.CRITICAL: Error thrown while running command "'doctrine:database' ':create'". Message: "Command "doctrine:database" is not defined.  Did you mean one of these?     doctrine:cache:clear-collection-region     doctrine:cache:clear-entity-region     doctrine:cache:clear-metadata     doctrine:cache:clear-query     doctrine:cache:clear-query-region     doctrine:cache:clear-result     doctrine:database:create     doctrine:database:drop     doctrine:ensure-production-settings     doctrine:mapping:convert     doctrine:mapping:import     doctrine:mapping:info     doctrine:migrations:current     doctrine:migrations:diff     doctrine:migrations:dump-schema     doctrine:migrations:execute     doctrine:migrations:generate     doctrine:migrations:latest     doctrine:migrations:list     doctrine:migrations:migrate     doctrine:migrations:rollup     doctrine:migrations:status     doctrine:migrations:sync-metadata-storage     doctrine:migrations:up-to-date     doctrine:migrations:version     doctrine:query:dql     doctrine:query:sql     doctrine:schema:create     doctrine:schema:drop     doctrine:schema:update     doctrine:schema:validate" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"doctrine:database\" is not defined.\n\nDid you mean one of these?\n    doctrine:cache:clear-collection-region\n    doctrine:cache:clear-entity-region\n    doctrine:cache:clear-metadata\n    doctrine:cache:clear-query\n    doctrine:cache:clear-query-region\n    doctrine:cache:clear-result\n    doctrine:database:create\n    doctrine:database:drop\n    doctrine:ensure-production-settings\n    doctrine:mapping:convert\n    doctrine:mapping:import\n    doctrine:mapping:info\n    doctrine:migrations:current\n    doctrine:migrations:diff\n    doctrine:migrations:dump-schema\n    doctrine:migrations:execute\n    doctrine:migrations:generate\n    doctrine:migrations:latest\n    doctrine:migrations:list\n    doctrine:migrations:migrate\n    doctrine:migrations:rollup\n    doctrine:migrations:status\n    doctrine:migrations:sync-metadata-storage\n    doctrine:migrations:up-to-date\n    doctrine:migrations:version\n    doctrine:query:dql\n    doctrine:query:sql\n    doctrine:schema:create\n    doctrine:schema:drop\n    doctrine:schema:update\n    doctrine:schema:validate at /home/cyril/Bureau/blog/vendor/symfony/console/Application.php:709)","command":"'doctrine:database' ':create'","message":"Command \"doctrine:database\" is not defined.\n\nDid you mean one of these?\n    doctrine:cache:clear-collection-region\n    doctrine:cache:clear-entity-region\n    doctrine:cache:clear-metadata\n    doctrine:cache:clear-query\n    doctrine:cache:clear-query-region\n    doctrine:cache:clear-result\n    doctrine:database:create\n    doctrine:database:drop\n    doctrine:ensure-production-settings\n    doctrine:mapping:convert\n    doctrine:mapping:import\n    doctrine:mapping:info\n    doctrine:migrations:current\n    doctrine:migrations:diff\n    doctrine:migrations:dump-schema\n    doctrine:migrations:execute\n    doctrine:migrations:generate\n    doctrine:migrations:latest\n    doctrine:migrations:list\n    doctrine:migrations:migrate\n    doctrine:migrations:rollup\n    doctrine:migrations:status\n    doctrine:migrations:sync-metadata-storage\n    doctrine:migrations:up-to-date\n    doctrine:migrations:version\n    doctrine:query:dql\n    doctrine:query:sql\n    doctrine:schema:create\n    doctrine:schema:drop\n    doctrine:schema:update\n    doctrine:schema:validate"} []
[2022-03-09T09:09:46.141024+01:00] console.CRITICAL: Error thrown while running command "doctrine:database:create". Message: "An exception occurred in the driver: could not find driver" {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)","command":"doctrine:database:create","message":"An exception occurred in the driver: could not find driver"} []
[2022-03-09T09:09:46.142166+01:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2022-03-09T09:33:54.972327+01:00] console.CRITICAL: Error thrown while running command "'doc:sch:upda' --force". Message: "An exception occurred in the driver: could not find driver" {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)","command":"'doc:sch:upda' --force","message":"An exception occurred in the driver: could not find driver"} []
[2022-03-09T09:33:54.972663+01:00] console.DEBUG: Command "'doc:sch:upda' --force" exited with code "1" {"command":"'doc:sch:upda' --force","code":1} []
[2022-03-09T09:35:16.761262+01:00] console.CRITICAL: Error thrown while running command "'doc:datab:crea'". Message: "An exception occurred in the driver: could not find driver" {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/cyril/Bureau/blog/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)","command":"'doc:datab:crea'","message":"An exception occurred in the driver: could not find driver"} []
[2022-03-09T09:35:16.763247+01:00] console.DEBUG: Command "'doc:datab:crea'" exited with code "1" {"command":"'doc:datab:crea'","code":1} []
